.github/workflows/wheels.yml

```
@@ -0,0 +1,60 @@
+name: Build and upload to PyPI
+
+# Alternatively, to publish when a (published) GitHub Release is created, use the following:
+on:
+  push:
+  pull_request:
+  release:
+    types:
+      - published
+
+jobs:
+  build_wheels:
+    name: Build wheels on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-22.10, macos-11]
```

```suggestion
        os: [ubuntu-22.04, macos-11]
```

I don't think 22.10 is available for github actions.  I think you this was a change to try and get a newer HDF5, but the `cibuildwheel` tool builds inside a docker image anyhow.

---

thanks, and double thanks for the inline changes, makes everything super easy to change